builtin/logical/pki/path_issue_sign.go

```diff
@@ -77,6 +77,11 @@ func buildPathIssue(b *backend, pattern string, displayAttrs *framework.DisplayA
 								Description: `Serial Number`,
 								Required:    true,
 							},
+							"not_before": {
```

Comment:
I hadn't thought about adding it to the output during issuance, I suppose that's fine :-)